components/brave_welcome_ui/components/screens/searchBox.tsx

```diff
@@ -21,11 +21,33 @@ interface Props {
   index: number
   currentScreen: number
   onClick: () => void
+  onChange: (event: React.ChangeEvent<HTMLSelectElement>) => void
+  isDefaultSearchGoogle: boolean
+  // TODO Pass in search options as an array of data and define specific type definition
+  searchProviders: Array<any>
+}
+
+interface State {
+  searchEngineSelected: boolean
 }
 
-export default class SearchEngineBox extends React.PureComponent<Props, {}> {
+export default class SearchEngineBox extends React.PureComponent<Props, State> {
+  constructor (props: Props) {
+    super(props)
+    this.state = {
+      searchEngineSelected: false
+    }
+  }
+
+  onChangeDefaultSearchEngine = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    this.setState({ searchEngineSelected: ((event.target.value) !== '') })
+    this.props.onChange(event)
+  }
+
   render () {
-    const { index, currentScreen, onClick } = this.props
+    const { index, currentScreen, onClick, isDefaultSearchGoogle, searchProviders } = this.props
+    const { searchEngineSelected } = this.state
+    const bodyText = isDefaultSearchGoogle ? `${getLocale('chooseSearchEngine')} ${getLocale('privateExperience')}` : getLocale('chooseSearchEngine')
```

Comment:
please add the locale for `privateExperience`. welcome page locale can be found in:

https://github.com/brave/brave-core/blob/master/browser/ui/webui/brave_webui_source.cc#L174
https://github.com/brave/brave-core/blob/master/components/resources/brave_components_strings.grd#L172